<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOperatorWindow</class>
 <widget class="QDialog" name="AddOperatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>173</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>243</width>
    <height>173</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>243</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ім'я</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Прізвище</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>По-батькові</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Пароль</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Логін</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="surnameField"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="passwordField"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="loginField"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="nameField"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="fathersNameField"/>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Відміна</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="addBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>surnameField</tabstop>
  <tabstop>nameField</tabstop>
  <tabstop>fathersNameField</tabstop>
  <tabstop>loginField</tabstop>
  <tabstop>passwordField</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
